assets/css/editor.scss

```diff
@@ -1,5 +1,6 @@
 // Import the woocommerce components stylesheet
 @import "~@woocommerce/components/build-style/style.css";
+@import "style.scss";
```

Comment:
@ryelle I added this for the server side rendering but might not be needed unless that goes ahead. Although there may be advantages to sharing styles between editor and frontend now we have custom HTML :)

Reply:
This shouldn't be necessary since the `style.css` is loaded in the editor already (it's a little counterintuitive, but `style` loads in both front-end and editor, while `editor` styles are just in the editor). Were you having trouble getting some styles to load?

Reply:
I did yes with your other PR. I'll remove it.